yapf: add page and Dutch translation (#5435)
tldr-pages/tldr: pages/common/yapf.md

# yapf

> > Python style guide checker

Reviewer: ```suggestion
> Python style guide checker.
```

> > Source code: https://github.com/google/yapf

Reviewer: We actually have a special format for links like this:
```suggestion
> More information: <https://github.com/google/yapf>.
```

Author: Sorry, it's my first addition to this project =D Noted for the future and I will commit your suggestions

Reviewer: No problem whatsoever! 🙂

> - Print out the diff that will occur after formatting.

Reviewer: ```suggestion
- Print out the diff that will occur after formatting:
```

> - Print out the diff that will occur after formatting:

Reviewer: ```suggestion
- Display the diff of the changes that would be made, without making them (dry-run):
```

> - Display the diff of the changes that would be made, without making them (dry-run):

Reviewer 2: Perhaps this will read better?
```suggestion
- Display a diff of the changes that would be made, without making them (dry-run):
```

`yapf --diff {{path/to/file}}`

> - Format the file in-place and and display diff:

Reviewer: ```suggestion
- Format the file in-place and display a diff of the changes:
```

> - Print out the formatted diff and make the changes in the file.

Reviewer: ```suggestion
- Print out the formatted diff and make the changes in the file:
```

> - Print out the formatted diff and make the changes in the file:

Reviewer: ```suggestion
- Format the file in-place and and display diff:
```

`yapf --diff --in-place {{path/to/file}}`

> - Format all Python files in a directory recursively in parallel:

Reviewer: ```suggestion
- Recursively format all Python files in a directory, concurrently:
```

> - Format all Python files in a directory recursivly (in pep8 style for example) in parallel.

Reviewer: ```suggestion
- Format all Python files in a directory recursivly (in pep8 style for example) in parallel:
```

> - Format all Python files in a directory recursivly (in pep8 style for example) in parallel:
> 
> `yapf --recursive --in-place --style pep8 --parallel {{path/to/directory}}`

Reviewer: ```suggestion
- Format all Python files in a directory recursively in parallel:

`yapf --recursive --in-place --style {{pep8}} --parallel {{path/to/directory}}`
```
